Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml

```diff
@@ -0,0 +1,530 @@
+      playbookName: File Enrichment - VMRay
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      MD5:
+        complex:
+          root: File
+          accessor: MD5
+      SHA1:
+        complex:
+          root: File
+          accessor: SHA1
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 0,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "19":
+    id: "19"
+    taskid: 36f34626-ee74-471a-8a5e-246ebe307101
+    type: playbook
+    task:
+      id: 36f34626-ee74-471a-8a5e-246ebe307101
+      version: -1
+      name: Valid SHA1
+      description: Test file enrichment with a valid SHA1 hash
+      playbookName: File Enrichment - VMRay
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "20"
+    scriptarguments:
+      MD5:
+        complex:
+          root: File
+          accessor: MD5
+      SHA1:
+        complex:
+          root: File
+          accessor: SHA1
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "20":
+    id: "20"
+    taskid: b3b711ba-592e-4c08-8247-1abbddae1008
+    type: playbook
+    task:
+      id: b3b711ba-592e-4c08-8247-1abbddae1008
+      version: -1
+      name: Unknown SHA1
+      description: Test file enrichment with an unknown SHA1 hash
+      playbookName: File Enrichment - VMRay
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "21"
+    scriptarguments:
+      MD5:
+        complex:
+          root: File
+          accessor: MD5
+      SHA1:
+        complex:
+          root: File
+          accessor: SHA1
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "21":
+    id: "21"
+    taskid: cc61427b-41ef-4924-88fe-4c4097ccede7
+    type: playbook
+    task:
+      id: cc61427b-41ef-4924-88fe-4c4097ccede7
+      version: -1
+      name: Invalid SHA1
```

Comment:
Please add the condition that the playbook won't stop on errors, as we discussed